### ISSUE TITLE
Error when using ng-repeat on an array of objects.

### ISSUE THREAD

**Issue by halloffame** [driftyco/ionic#2466]
http://codepen.io/anon/pen/BkgwE

It should loop through just once, but instead it keeps going back and looping through the array again and again, until it stops because it reaches too many digest iterations. When i stop including 'ionic' into the app it starts working normally.

Here is the error I am getting.
```
Uncaught Error: [$rootScope:infdig] 10 $digest() iterations reached. Aborting!
Watchers fired in the last 5 iterations: [["fn: $watchCollectionWatch; newVal: 9; oldVal: 8","fn: function (context) {
          .......... lots of stuff .........
http://errors.angularjs.org/1.2.25/$rootScope/infdig?p0=10&p1=%5B%5B%22fn%3…20%7D%3B%20newVal%3A%20%5C%22123%5C%22%3B%20oldVal%3A%20undefined%22%5D%5D ionic.bundle.js:20433
```

**Comment by Ionitron** [comment]
Greetings @halloffame!

I've closed this issue because my sensors indicated it was old and inactive, and may have already been fixed in recent versions of Ionic. However, if you are still experiencing this issue, please feel free to reopen it and include any examples and other necessary information, so that we can look into it further.

Thank you for allowing me to assist you.

**Comment by halloffame** [comment]
Thanks! It looks like this was fixed in 1.0.0-beta.14
http://jsfiddle.net/556GL/36/

**Comment by ituhin** [comment]
i am getting this is safari console from simulator ==> 
Error: [$rootScope:infdig] 10 $digest() iterations reached. Aborting!
Watchers fired in the last 5 iterations: []
http://errors.angularjs.org/1.3.13/$rootScope/infdig?p0=10&p1=%5B%5


$ ionic info
Cordova CLI: 5.2.0
Ionic CLI Version: 1.6.4
Ionic App Lib Version: 0.3.8
ios-deploy version: 1.7.0 
ios-sim version: 4.1.1 
OS: Mac OS X Yosemite
Node Version: v0.12.7
Xcode version: Xcode 7.0 Build version 7A176x
